apps/src/code-studio/assets/SpriteManagementDirectory.jsx

```diff
@@ -1,7 +1,26 @@
 import React from 'react';
+import Button from '@cdo/apps/templates/Button';
+import color from '@cdo/apps/util/color';
+import {moveDefaultSpriteMetadataToProduction} from '@cdo/apps/assetManagement/animationLibraryApi';
 
 export default class SpriteManagementDirectory extends React.Component {
+  state = {
```

Comment:
since we're introducing state management to this component (rather than adding to existing functionality), we should use hooks and a functional component here!